src/modules/featureInfo/components/FeatureInfoPanel.js

```diff
@@ -31,6 +32,7 @@ export class FeatureInfoPanel extends AbstractMvuContentPanel {
 		this._translationService = TranslationService;
 
 		this.observe(store => store.featureInfo.current, current => this.signal(Update_FeatureInfo_Data, [...current]));
+		this.observe(state => state.media, media => this.signal(Update_IsPortrait, { isPortrait: media.portrait }));
```

Comment:
Would suggest the same approach as in line34, we broadcast just the value, not an object:
```
this.observe(state => state.media, media => this.signal(Update_IsPortrait, media.portrait ));
```